web-common/src/features/dashboards/dashboard-stores.ts

```diff
@@ -107,10 +74,10 @@ export interface MetricsExplorerEntity {
 
   // This is the sort type that will be used for the leaderboard
   // and dimension detail table. See SortType for more details.
-  dashboardSortType: SortType;
+  dashboardSortType: DashboardState_LeaderboardSortType;
```

Comment:
@AdityaHegde -- for readability, can you change the imports for these renamed enums to `import {DashboardState_LeaderboardSortType as SortType} from ...` so we don't have these long ugly strings all over the place?

(and likewise of course for `DashboardState_LeaderboardSortDirection`)

Please update this in all files, thanks!

Reply:
that would also reduce the merge conflicts for #2957 and #2951

Reply:
I went ahead and made this change in this commit, https://github.com/rilldata/rill/pull/2932/commits/a420c865d4424f29e2554f85dfffe26455223c73, please double check it to make sure that looks ok to you :-)

Reply:
Thanks!